spec/requests/user_status_controller_spec.rb

```diff
@@ -38,38 +38,67 @@
       it "sets user status" do
         status = "off to dentist"
         status_emoji = "tooth"
-        put "/user-status.json", params: { description: status, emoji: status_emoji }
+        ends_at = DateTime.parse("2100-01-01 18:00")
+
+        put "/user-status.json", params: {
+          description: status,
+          emoji: status_emoji,
+          ends_at: ends_at
+        }
```

Comment:
We probably want to assert for the response code to make the error message more helpful if the request fails.

Reply:
Sure, that'll be good, I've added assertions.